__performance__/README.md

```diff
@@ -0,0 +1,143 @@
+
+TARGET_URL is used to set the absolute test path.
+
+### Stress
+
+> The purpose of stress testing is to assess the availability and stability of the system under heavy load. - [k6](https://k6.io/docs/test-types/stress-testing)
+
+`TARGET_BASE_URL` is used to set the base test path. It is recommended to only run this against test environments which use the `prod-sample` modules.
+
+The following paths will be expected to be available:
+- /success
+- /healthy-frank
+- /demo/ssr-frank
+- /demo/ssr-frank
+- /demo/needy-frank?api=https://fast.api.frank/posts
+- /demo/needy-frank?api=https://slow.api.frank/posts
+
+### Spike
+
+> Spike test is a variation of a stress test, but it does not gradually increase the load, instead it spikes to extreme load over a very short window of time. While a stress test allows the SUT (System Under Test) to gradually scale up its infrastructure, a spike test does not. [k6](https://k6.io/docs/test-types/stress-testing#spike-testing)
+
+
+`TARGET_BASE_URL` is used to set the base test path. It is recommended to only run this against test environments which use the `prod-sample` modules.
+
+The following paths will be expected to be available:
+- /success
+- /healthy-frank
+- /demo/ssr-frank
+- /demo/ssr-frank
+- /demo/needy-frank?api=https://fast.api.frank/posts
+- /demo/needy-frank?api=https://slow.api.frank/posts
+
+
+## Running against prod-sample
+
+### Setup
+
+From within the one-app directory first start the one-app prod-sample:
+
+```bash
+npm run start:prod-sample
+```
+
+Wait for `🌎 One App server listening on port 8443` to appear in the logs to indicate the one-app server has successfully started.
+
+> `npm run start:prod-sample` creates the `prod-sample_one-app-at-test-network` network which is required for the performance tools to start when running against prod-sample.
+
+After prod-sample has successfully started you will need to start the performance test suite infrastructure in a new terminal window, navigate to `one-app/__performance__` and run the following:
+
+```bash
+docker-compose -f docker-compose.yml -f docker-compose.prod-sample.yml up --abort-on-container-exit influxdb grafana prometheus
+```
+
+You can now view the [Grafana metrics on localhost:3030](http://localhost:3030/d/tDGvrq7Mz/one-app-performance)
+
+### Run the smoke test
+
+In another new window from within `one-app/__performance__` run the following:
+
+```bash
+docker-compose -f docker-compose.yml -f docker-compose.prod-sample.yml run k6 run --insecure-skip-tls-verify /scripts/smoke.js
+```
+
+Each file under `__perforamance__/scripts` correlates to a different test:
+
+Smoke - `/scripts/smoke.js`
+Stress - `/scripts/stress.js`
+Load - `/scripts/load.js`
+Soak - `/scripts/soak.js`
+
+To run a test other than the smoke test replace `/scripts/smoke.js` with the corresponding
+file path outlined above.
+
+Once the test has finished running you can use Grafana to share a snapshot
```

Comment:
Worth adding the caveat that you have to reload grafana to see the results of the test?

Reply:
you should not have to, there is a little button at the top right that triggers another poll